test/12_missingarray.jl

```diff
@@ -1197,4 +1197,10 @@ end
     @test y == a
 end
 
+@testset "sizehint!" begin
```

Comment:
Sorry, actually 11_array.jl and 12_missingarray.jl follow exactly the same structure, the only difference being that the latter tests missing values. But in the present case missing values shouldn't matter much, so it's probably better to just put these in 13_arraycommon.jl (and maybe not test with missing values at all).

(That organization is mostly historical...)

Reply:
done